### PR TITLE
Update basics-solidity.md

### PR DESCRIPTION
**Pull request by annu12340**
Improved the documentation

### CODE REVIEW

File: docs/home/blockchain-basics/basics-solidity.md [hunk 1]
```diff
@@ -28,6 +34,11 @@ Essentially the following points need to be kept in mind always:
 - Reading data is free
 - Each write operation has a cost associated with it - you pay it with ether. The cost is measured in gas.
 
+#### Tools for Working With Solidity
+- Truffle: Ethereum blockchain framework
```
Comment (itzmeanjan) [inline docs/home/blockchain-basics/basics-solidity.md:38]: Truffle is more like a dApp building framework. So I'd prefer to go with `Truffle: DApp development framework`.

**Comment by itzmeanjan** [comment]
Hello @annu12340 , good work ! Can you please check [this](https://github.com/maticnetwork/matic-docs/pull/205#discussion_r458719084), and make necessary change ? Then we'll be good to merge your changes.

File: docs/home/blockchain-basics/basics-solidity.md (hunk 1 above)
Comment (annu12340) [reply docs/home/blockchain-basics/basics-solidity.md]: I have made the requested change

Comment (itzmeanjan) [reply docs/home/blockchain-basics/basics-solidity.md]: Thanks :+1:

**Pull request merged**